nucypher/cli/actions/select.py

```diff
@@ -93,64 +108,84 @@ def select_client_account(emitter,
 
     enumerated_accounts = dict(enumerate(signer.accounts))
     if len(enumerated_accounts) < 1:
-        emitter.echo(NO_ETH_ACCOUNTS, color='red', bold=True)
+        emitter.echo(NO_ETH_ACCOUNTS, color="red", bold=True)
         raise click.Abort()
     elif len(enumerated_accounts) == 1:
         # There are no choices if there is only one available address.
         return enumerated_accounts[0]
 
     # Display account info
-    headers = ['Account']
+    headers = ["Account"]
```

Comment:
Is it a linter producing these extra quotation changes...might want to disable some of these - makes the PR bigger than necessary...